packages/react-art/src/ReactARTHostConfig.js

```diff
@@ -346,6 +346,10 @@ export function getCurrentEventPriority() {
   return DefaultEventPriority;
 }
 
+export function shouldAttemptEagerTransition() {
+  return false;
```

Comment:
React ART is a secondary renderer that is always embedded in React DOM. So it could be that clicking the Back button triggers an update in a React ART tree and so I think it should have the same logic. Not a huge deal though since almost no one uses it and right now the bridge between React DOM and React ART is a class component that triggers the update in the commit phase, lol:

https://github.com/facebook/react/blob/353ceed439b942096baafb594b794941037f628b/packages/react-art/src/ReactART.js#L87

Reply:
I just noticed ART is also treating all events as default priority, so it seems it's already different from ReactDOM, https://github.com/facebook/react/blob/79294e914b889b7d65bc6ffe86dd7d69502a7f4a/packages/react-art/src/ReactFiberConfigART.js#L345-L347

and there is no microtask which this PR depends on
https://github.com/facebook/react/blob/79294e914b889b7d65bc6ffe86dd7d69502a7f4a/packages/react-art/src/ReactFiberConfigART.js#L245